src/index.js

```diff
@@ -93,7 +95,15 @@ export default class TrackingWallet {
     if (isTestUser()) return;
 
     if (hasWindow()) {
-      window.analytics.page();
+      if (isAgent()) {
+        window.analytics.page({ userRole: AGENT_VALUE });
+      }
+      if (isInternalUser()) {
+        window.analytics.page({ userRole: CW_INTERNAL_USER });
+      }
+      if (isTestUser()) {
+        window.analytics.page({ userRole: CW_TEST });
+      }
```

Comment:
What do you think about:

```suggestion
      const pageProps = {};

      if (isAgent()) {
        pageProps.userRole = AGENT_VALUE;
      }
      if (isInternalUser()) {
        pageProps.userRole = CW_INTERNAL_USER;
      }
      if (isTestUser()) {
        pageProps.userRole = CW_TEST;
      }

      window.analytics.page(pageProps);
```

Having just one analytics.page call that is less nested inside "ifs" makes It easier to track when and when not is being called

Reply:
much better 👍🏻